### PR TITLE
Release 3.0

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

**Comment by BenFradet** [comment]
@JanOuborny could you review? This is what I published btw.

File: RiotSharp.AspNetCore/RiotSharp.AspNetCore.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (JanOuborny) [inline RiotSharp.AspNetCore/RiotSharp.AspNetCore.csproj]: Why do you changing this to 2.0? It just lowers the compatibility.

File: RiotSharp.Test/appsettings.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "ApiKey": "API_KEY",
+  "ApiKey": "RGAPI-b2402d23-fb1b-4bdc-8e95-397b1a8f73cc",
```
Comment (JanOuborny) [inline RiotSharp.Test/appsettings.json]: Forgot your key here.

Comment (BenFradet) [reply RiotSharp.Test/appsettings.json]: it already expired no worries

**Comment by BenFradet** [comment]
@JanOuborny updated 👍

**Comment by BenFradet** [comment]
@JanOuborny can you review?

File: README.md [hunk 3]
```diff
@@ -168,7 +168,7 @@ foreach (var service in shardStatuses.Services)
 You can find a list of all the available operations in [StatusRiotApi in the documentation](http://benfradet.github.io/RiotSharp/api/RiotSharp.StatusRiotApi.html).
 
 
-For a full description check the [RiotSharpExample](RiotSharpExample) or [RiotSharpTest](RiotSharpTest) projects.
+For a full description check the [RiotSharpTest](RiotSharpTest) project.
```
Comment (JanOuborny) [inline README.md]: I don't think the unit test project is a usefull description for new users. You might want to remove this line.

File: RiotSharp/RiotSharp.csproj.user [hunk 4]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (JanOuborny) [inline RiotSharp/RiotSharp.csproj.user]: You might want to ignore this file.

File: README.md (hunk 3 above)
Comment (BenFradet) [reply README.md]: I removed the example project so this is what's left, either that and/or the `IRiotApi` interface.

Why did I remove the example project? It presents only a small subset of the api and I don't think it's that useful unfortunately.

**Pull request merged**